### DISCUSSION TITLE
Choice for users to decide Save files location

### DISCUSSION THREAD

**Discussion by mrtawsiti3** [open-goal/jak-project#3641] [Ideas]
For example, all the save files from jak games goes to fixed internal location.

I would like to move save files location from internal to external location for etc. MicroSD Card and would like it to stay there and function from there.

**Comment by Zedb0T** [comment]
This is possible with this PR -> https://github.com/open-goal/jak-project/pull/3477 

You just need to pass --config-path to gk
